src/common/offline_transformations/include/compress_quantize_weights.hpp

```diff
@@ -63,30 +62,3 @@ class ov::pass::CompressQuantizeWeights : public ov::pass::MatcherPass {
     OPENVINO_RTTI("CompressQuantizeWeights", "0");
     CompressQuantizeWeights();
 };
-
-/*
-   if zero_point == 0 we can eliminate Subtract from following dequantization subgraph:
-
-                                +-----------------+
-                                |    Constant     |
-                                | (low precision) |
-                                +-----------------+
-                                        |
-                                        v
-                                +------------------+
-                                |     Convert      |
-                                |  (to high prec)  |
-                                +------------------+
-                                        |
-                                        v
-                  +----------+    +------------+
-                  |zero point|--->|  Subtract  |
-                  +----------+    +-----+------+
-                                        |
-                                        v
-*/
```

Comment:
could you add this description to CompressQuantizeWeights transformation?

Reply:
done